.github/CONTRIBUTING.rst

```diff
@@ -31,6 +31,9 @@ also suggestions on how you can most effectively help the project.
 Please be aware that our workflow does deviate slightly from the typical GitHub
 project. Details on how to properly submit a pull request are covered in
 `Lifecycle of a Pull Request <https://cpython-devguide.readthedocs.io/pullrequest.html>`_.
+One key point is to keep comments on GitHub to only those related to the reviewing
+the code in the pull request. All other discussions -- e.g. about the issue being
```

Comment:
It might be helpful to clarify what "other discussions" mean.  

    All other discussions, such as evaluating if it issue reported is accurate, if the problem is worth fixing or approaches that should be taken to fix the issue, should happen on bugs.python.org.